frontend/src/css/app.sass

```diff
@@ -76,8 +76,6 @@
 .danger
```

Comment:
*arbitrary comment location*

Hey @garyghayrat, overall this is looking good, however I'm observing the following behavior with regards to advanced mode: if the user has never registered stealth keys before, but turns on advanced mode, it now fetches and scans *all announcements*. Why is this the case? We should only need to scan all blocks if the user supplies a private key (because now we don't know when that was registered) or if the user specifies a block range (in which case we should respect it). Am I missing something?

Reply:
Updated here: https://github.com/ScopeLift/umbra-protocol/pull/571/commits/ac938e6ec365de35ee3c76035eb169357c5a601a

Reply:
Ok @garyghayrat, now I'm seeing the opposite problem:

* I connect with an account that *has* completed setup, and *does* have past transactions in its history
* I turn on advanced mode and scan
* It completes immediately and it shows no transactions
* I turn advanced mode off, refresh and scan
* It scans much longer and shows the past transactions

See the video below:

![umbra-advanced-no-scan](https://github.com/ScopeLift/umbra-protocol/assets/1479802/85cb9a29-3357-4239-8387-82b5685dce06)

Reply:
Ok, this should fix it: https://github.com/ScopeLift/umbra-protocol/pull/571/commits/dd8e70078af457c73b67a3589e92e25df5510809